pulp_container/app/settings.py

```diff
@@ -8,3 +8,8 @@
 
 # The number of allowed threads to sign manifests in parallel
 MAX_PARALLEL_SIGNING_TASKS = 10
+
+# Set max payload size for non-blob container artifacts (manifests, signatures, etc).
+# This limit is also valid for docker manifests, but we will use the OCI_ prefix
+# (instead of ARTIFACT_) to avoid confusion with pulpcore artifacts.
+OCI_PAYLOAD_MAX_SIZE = 4_000_000
```

Comment:
```suggestion
# This limit is also valid for docker manifests
MAX_MANIFEST_BODY_SIZE = 4_000_000
```
Is it 4MB?
```
>>> 1 << 20
1048576
>>> 1024 * 1024
1048576
>>> 4 * 1 << 20
4194304
>>> 4 * 1024 * 1024
4194304
```
ref: https://github.com/distribution/distribution/blob/d0eebf3af4fc1d5c0287e5af61147403ccb78ec2/registry/handlers/manifests.go#L29, https://github.com/containers/image/blob/8d792a4a930c36ae3228061531cca0958ba4fe0a/internal/iolimits/iolimits.go#L20